src/services/profile.service.js

```diff
@@ -0,0 +1,38 @@
+import models from '../database/models';
+
+const { user } = models;
+
+/**
+ * @description class ProfileService handles everything regarding to profile
+ */
+export default class ProfileService {
+  /**
+   * @param{number} requestedProfile
+   * @returns{object} user
+   */
+  static readProfile = async (requestedProfile) => {
```

Comment:
could you rename this arrow function to something more descriptive like `getUserProfile`

Reply:
Yes sure, the action is taken successfully